Scraper.py

```diff
@@ -47,6 +47,13 @@ def _timestamp(self) -> str:
         """
         # There's some delay in the API so you need to delay the timestamp a little.
         # With about 20 minutes of hair pulling "why is the api not returning anything!!!!" frustration I found that 45 seconds was the smallest round-ish number that got returns.
-        time = datetime.now(pytz.utc) - timedelta(seconds=45)
+        # We also scrape back 5 minutes to check for long calls.
+        time = datetime.now(pytz.utc) - timedelta(seconds=(self.lookback + 45))
+
+        # If it's the first check try to get nothing
+        if self.firstCheck:
+            time = datetime.now(pytz.utc)
+            self.firstCheck = False

```

Comment:
So simple! Very nice :slightly_smiling_face:

Reply:
Sometimes quick and dirty works!